data/catalogs/changelog.rst

```diff
@@ -24,6 +24,90 @@ Added
+- prefixed 'hydro' for all HydroSHEDS datasets
+- added the epoch of the dataset to the name of the dataset
+
+See table below for mapping of old and new names:
+
++--------------------------------+---------------------------+
+| Old name                       | New name                  |
++================================+===========================+
+| basin_atlas_level12_v10        | hydro_basin_atlas_level12 |
++--------------------------------+---------------------------+
+| eobs_v..                       | eobs                      |
++--------------------------------+---------------------------+
+| eobs_orography_v..             | eobs_orography            |
++--------------------------------+---------------------------+
+| lake_atlas_pol_v10             | hydro_lake_atlas_pol      |
++--------------------------------+---------------------------+
+| river_atlas_v10                | hydro_river_atlas         |
++--------------------------------+---------------------------+
+| ghs_pop_2015_54009_v2019a      | ghs_pop_2015	             |
++--------------------------------+---------------------------+
+| ghs_smod_2015_54009_v2019a     | ghs_smod_2015             |
++--------------------------------+---------------------------+
+| glofas_era5_v31                | glofas_era5               |
++--------------------------------+---------------------------+
+| guf_bld_2012                   | guf_bld_2012              |
++--------------------------------+---------------------------+
+| rivers_lin2019_v1              | hydro_rivers_lin2019      |
++--------------------------------+---------------------------+
+| SM2RAIN_ASCAT_monthly_025_v1.4 | SM2RAIN_ASCAT_monthly_025 |
++--------------------------------+---------------------------+
+| SM2RAIN_ASCAT_monthly_05_v1.4  | SM2RAIN_ASCAT_monthly_05  |
++--------------------------------+---------------------------+
+| vito 					         | vito_2015                 |
++--------------------------------+---------------------------+
+| vito_2015_v2.0.2			   	 | vito_2015                 |
++--------------------------------+---------------------------+
+| vito_2016_v3.0.1               | vito_2016                 |
++--------------------------------+---------------------------+
+| vito_2017_v3.0.1               | vito_2017                 |
++--------------------------------+---------------------------+
+| vito_2018_v3.0.1               | vito_2018                 |
++--------------------------------+---------------------------+
+| vito_2019_v3.0.1               | vito_2019                 |
++--------------------------------+---------------------------+
+| esa_worldcover 			     | esa_worldcover_2020       |
++--------------------------------+---------------------------+
+| corine 					     | corine_2018               |
++--------------------------------+---------------------------+
+| globcover 				     | globcover_2009            |
++--------------------------------+---------------------------+
```

Comment:
I actually see that you added the year of the landuse map to also in a way match vito. For data like globcover there was really only one year so I dont know if we need to add. For esa_worldcover the 2021 map is actually under a separate version number. And corine I think so far also one map only.
So in a way I don't really mind if you prefer to leave the years after all or not, just make sure they are the same in artifact_data as well.

Reply:
I removed the years in the name of the corine, esa_worldcover, and globcover datasets. It makes more sense if these datasets are from one year only.